Controllers/errorController.js

```diff
@@ -1,31 +1,31 @@
 const sendError = (err, res) => {
-  console.log(err.message);
-  if (err.message === "Email must be unique") {
+  console.log(err.message)
+  if (err.message === 'Email must be unique') {
     res.status(400).json({
-      status: "fail",
-      message: "Selected email is already in use",
-    });
-  } else if (err.message === "Username must be unique") {
+      status: 'fail',
+      message: 'Selected email is already in use'
+    })
+  } else if (err.message === 'Username must be unique') {
     res.status(400).json({
-      status: "fail",
-      message: "Selected username is already in use",
-    });
-  } else if(err.message === 'Incorrect email or password'){
+      status: 'fail',
+      message: 'Selected username is already in use'
+    })
+  } else if (err.message === 'Incorrect email or password') {
     res.status(400).json({
-      status: "fail",
-      message: "Email or password is incorrect",
-    });
+      status: 'fail',
+      message: 'Email or password is incorrect'
+    })
   } else {
     res.status(400).json({
```

Comment:
Check your response codes, code 400 should only be used if the client sends a bad request.